Parsers/ASimWebSession/Parsers/imWebSession.yaml

```diff
@@ -1,103 +1,104 @@
-Parser:
```

Comment:
@jaimeesc change is not clear. Can you please undo other change so only new change shows.

Reply:
Thank you for that! Appreciate the help.

I'm not sure why the diff shows up this way. I'll try rolling back the file to the current master state and will then commit a new change with my changes.

Reply:
I'm not sure what I can do with this file. Here's a diff of the file compared to the master branch. I just added a comma to the end of the previous line, then added a line for vimWebSessionSonicWallFirewall.

![image](https://github.com/Azure/Azure-Sentinel/assets/42151366/9e92a3c2-5f14-4ad5-b0dc-a37071342434)

![image](https://github.com/Azure/Azure-Sentinel/assets/42151366/de1f316a-6626-4818-969e-6568e6071453)